### ISSUE TITLE
v.4.0.2 broke compatibility with node 8

### ISSUE THREAD

**Issue by vierno** [yargs/y18n#125]
When using node <10, it's no longer possible to install y18n@4:

```
node --version
v8.17.0

yarn add y18n@4.0.2
yarn add v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error y18n@4.0.2: The engine "node" is incompatible with this module. Expected version ">=10". Got "8.17.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

v4.0.1 works fine:
```
yarn add y18n@4.0.1
yarn add v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
```

Was it an intended change of compatibility?

**Comment by bcoe** [comment]
fixed by 4.0.3:

```
npm notice 
npm notice 📦  y18n@4.0.3
npm notice === Tarball Contents === 
npm notice 731B  LICENSE     
npm notice 5.3kB index.js    
npm notice 889B  package.json
npm notice 1.3kB CHANGELOG.md
npm notice 2.8kB README.md   
npm notice === Tarball Details === 
npm notice name:          y18n                                    
npm notice version:       4.0.3                                   
npm notice package size:  4.3 kB                                  
npm notice unpacked size: 11.0 kB                                 
npm notice shasum:        b5f259c82cd6e336921efd7bfd8bf560de9eeedf
npm notice integrity:     sha512-JKhqTOwSrqNA1[...]Fmlb/U4UZ4hJQ==
npm notice total files:   5                                       
npm notice 
+ y18n@4.0.3
bencoe-macboo
```

**Comment by bcoe** [comment]
@glasser @vierno I've just started testing automation for back porting security fixes, which makes the process less terrible.
